libs/langchain/langchain/llms/vertexai.py

```diff
@@ -194,9 +223,41 @@ async def _acall(
             The string generated by the model.
         """
         return await asyncio.wrap_future(
-            self._get_task_executor().submit(self._predict, prompt, stop)
+            self._get_task_executor().submit(self._call, prompt, stop)
         )
 
+
+class VertexAIModelGarden(_VertexAIBase, LLM):
+    """Large language models served from Vertex AI Model Garden."""
+
+    client: "PredictionServiceClient" = None  #: :meta private:
+    project: str
```

Comment:
is this already defined in _VertexAIBase

Reply:
It's optional in `_VertexAIBase`. But that's OK, I added a check that raises a ValueError is it's None.